modules/models/src/main/java/org/eclipse/xpanse/modules/models/resource/CloudServiceProvider.java

```diff
@@ -21,7 +21,7 @@
 public class CloudServiceProvider {
 
     @NotNull
-    @Schema(description = "The Cloud Service Provider. valid values: aws, azure, alibaba, huawei")
+    @Schema(description = "The Cloud Service Provider. valid values: aws, azure, alicloud, huawei, flexibleEngine")
```

Comment:
we had actually discussed to remove the name from the description. Looks like it is still there in this one place. So we can remove here too.
The list of valid CSPs is anyways in the enum.

Reply:
done